backends/tfhe-cuda-backend/cuda/include/integer/integer_utilities.h

```diff
@@ -4153,6 +4153,771 @@ template <typename Torus> struct int_comparison_buffer {
   }
 };
 
+template <typename Torus> struct int_sub_and_propagate {
```

Comment:
This and the int_bitop_buffer are just changing places right? Could you put them where they were before to avoid unnecessary conflicts later on please?